.github/workflows/flake8.yml

```diff
@@ -0,0 +1,23 @@
+name: Check Code Style - FLAKE8
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install Dependencies
+        run: |
+          # These packages are installed in the base environment but may be older
+          # versions. Explicitly upgrade them because they often create
+          # installation problems if out of date.
+          python -m pip install --upgrade pip setuptools numpy
+
+          pip install flake8
+      - name: Run flake8
+        run: |
+          flake8
```

Comment:
excellent to add this!  I clearly forgot to do it with the last PR

the helpers modules are the only thing affected (my addition).

so the question is how to resolve the CI error for flake8?  The options are:
1. add it to issues and take care of it when I fix the bug in helpers.overscan
2. fix it now

Option 2, fix it now depends on who can do what and who is willing to wait. 
1. I can checkout this branch fix flake8
2. we (I or someone else) can make a different branch that gets merged now and then Phil can pull in those changes
3. Phil fix in place (but I don't know if he is available to do that on the timeline we need)

@mrakitin are you hoping for this to be done mid-next week (Nov 8)?  Depending on that and who we agree does what, I think we can move forward without make a mess of two people trying to fix the same thing.

Reply:
only can click "resolve conversation".  If I do that now, my comment collapses.  Sorry about that.

Reply:
I think it's not that time-pressing. We can resolve the linting/styling issues later as the release of v0.2.1 is already on PyPI, so it unblocks our work for raydata.

Reply:
great.  added issue so will resolve here